### PR TITLE
Update core dependencies

### PR DESCRIPTION
**Pull request by jdillon**
sisu/guice/guava & related.

### CODE REVIEW

**Comment by jdillon** [comment]
will link to grid build on success.

**Comment by jdillon** [comment]
https://builds.sonatype.org/job/nexus-oss-its-feature/470/ lnx=SUCCESS win=FAILED 

1 failure in DefaultFSLocalRepositoryStoragePerformance2IT.testRetieveItemWithLastAccessUpdate, which I think this should sort out:

https://github.com/sonatype/nexus/pull/622

IIUC this is unrelated to the dependency changes.

**Comment by peterlynch** [comment]
+1

File: nexus-runtime-platform/pom.xml [hunk 1]
```diff
@@ -23,115 +23,114 @@
     <relativePath />
   </parent>
 
+  <!--
+  FIXME: This does not actually provide deps on the Nexus runtime, but only on the sisu/guice/plexus and basic support libraries.
```
Comment (peterlynch) [inline nexus-runtime-platform/pom.xml:27]: nexus runtime means ' what common bits does nexus runtime need at runtime'

Comment (jdillon) [inline nexus-runtime-platform/pom.xml:27]: Fair enough, though I don't think that really warrants a separate top-level pom just for that... but perhaps its needed by something in pro?

Comment (cstamas) [inline nexus-runtime-platform/pom.xml:27]: Yes. Its used to align sisu and other deps as we had problems with that
before

Thanks,
~t~ mobile

Comment (cstamas) [inline nexus-runtime-platform/pom.xml:27]: Forgot to add: align them between oss and pro. Just as JD said

Thanks,
~t~ mobile

**Comment by jdillon** [comment]
Another win-only failure:

https://builds.sonatype.org/job/nexus-oss-its-feature/471/

org.sonatype.nexus.proxy.walker.FixedRateWalkerThrottleControllerTest.testDoesItHelpAtAll

**Pull request merged**